orchestration/values.schema.json

```diff
@@ -25,7 +25,16 @@
         "extractSize": { "type": "string" }
       },
       "required": ["storageClass", "downloadSize", "extractSize"]
+    },
+    "cron": {
+      "type": "object",
+      "properties": {
+        "enable": { "type": "boolean" },
+        "schedule": { "type": "string" },
```

Comment:
see if you can validate via regex

Reply:
Turns out crontabs are more complex than I thought, I'm going to punt on this for now.